scss/decanter.scss

```diff
@@ -1,20 +1,39 @@
-@charset "UTF-8";
+@charset 'UTF-8';
 
+///
+// Load Order
 //
-// DECANTER
+// 1. Pre-vendor settings
+// 2. Vendors
+// 3. Post-vendor settings
+// 4. Utilities (functions, mixins, extends)
+// 5. Core Elements
+// 6. Components
 //
 
-// Vendor
+// 1. Pre-vendor settings.
+@import 'variables/pre-vendor';
```

Comment:
Adding a pre-vendor option is important as bourbon/neat/bitters can require some variables to be set BEFORE they are imported.